ls1intum/Artemis
# `Quiz exercises`: Improve visibility of quiz started warning

### Checklist
#### General
#### Client

### Motivation and Context
Closes #4824

### Description
This PR moves the warning about stated quizzes to the top of the side.
It does not change the grey background since this implementation seems a bit hacky to me, it's a big div layered over the whole side, blocking every input. Changing this would require more refactoring and is out of scope of this small fix. 

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Quiz exercise

1. Start the quiz
2. Open the edit-view (by clicking on the id)
3. Make sure that the warning is on top of the page.

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
![grafik](https://user-images.githubusercontent.com/26540346/159042194-52ac4734-3448-4288-ae7f-667ad667ca41.png)
